scripts/config/allo.config.ts

```diff
@@ -95,10 +95,10 @@ export const alloConfig: DeployParams = {
     percentFee: 0,
     baseFee: 0,
   },
-  // Arbitrum Goerli
-  421613: {
-    owner: "0xE64c2171bB7c7412eb2ad3c4c027C499FEf345DC",
-    treasury: "0xE64c2171bB7c7412eb2ad3c4c027C499FEf345DC",
+  // Arbitrum Sepolia
+  421614: {
+    owner: "0x8C180840fcBb90CE8464B4eCd12ab0f840c6647C",
+    treasury: "0x8C180840fcBb90CE8464B4eCd12ab0f840c6647C",
```

Comment:
Is this a reason we use a different EOA than we what had used on Arb Goeri ?

Reply:
There were a few different ones so we used the one with funds and was already present in the code.